code-challenges.test.js

```diff
@@ -63,12 +96,21 @@ var middleLetters1 = "alpha"
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
 
-
+//use .split and .indexOf
 
 
 
 // b) Create the function that makes the test pass.
-
+describe("when a string of letters is called", () => {
+  it("returns the middle leter if the word has an odd amount of numbers and two middle letters if even", () => {
+    const str = (array)
+    const middleLetters1 = (array) => {
+      str.split(array) = array.indexOf(5)
+      return array
+    }
+expect("returns middle letter if odd and both middle letters if even")
```

Comment:
`expect()` takes the actual value returned from a function then you can use a matcher method like `.toEqual()` to compare the value to some expected outcome